client/components/tinymce/plugins/simple-payments/style.scss

```diff
@@ -8,6 +8,7 @@
 		padding: 0;
 		display: flex;
 		flex-direction: column;
+		max-width: none;
```

Comment:
I was actually tempted to do that, but other modals like (media library, image detail etc.) are now 90% (I think this is a recent change.). I wanted to research why the change was introduced before I add a custom style to it.

Can we go ahead with this for now, and I'll suggest to remove the max-height rule globally?